src/status_im/ui/screens/communities/membership.cljs

```diff
@@ -10,13 +10,13 @@
 (def options {constants/community-on-request-access
               {:title       :t/membership-approval
                :description :t/membership-approval-description}
-              constants/community-invitation-only-access
-              {:title       :t/membership-invite
-               :description :t/membership-invite-description}
+              ;; constants/community-invitation-only-access
```

Comment:
Is this supposed to be commented?

Reply:
Yes. This can be a feature we bring back in future, but as for now will be hidden from users.